src/courses/forms.py

```diff
@@ -49,6 +49,17 @@ class Meta:
     #         # widgets = {'user': forms.HiddenInput()}
 
 
+class CourseStudentStaffForm(CourseStudentForm):
+
+    class Meta:
+        model = CourseStudent
+        fields = '__all__'
+        exclude = ['user', 'active'] 
```

Comment:
I think only one of `fields` or `exclude` are required.  exclude is essentially "all except these ones". Please verify

Reply:
removing fields didnt change anything